docs/operations/testing.md

```diff
@@ -4,4 +4,117 @@ title: Testing
 
 # Testing
 
-Documentation coming soon
+The GraphiQL playground integrated with Strawberry available at
+[http://localhost:8000/graphql](http://localhost:8000/graphql) can be a good place to
```

Comment:
This is only when using Strawberry in a specific way (i.e. with the `strawberry server` command). Maybe we should make that clear

Reply:
I add a parenthesis `(if you run the schema
with `strawberry server`)` ... didn't come to mind a better way to write it... 😪

Reply:
I think it's good enough for now :+1: